packages/eslint-plugin-pf-codemods/index.js

```diff
@@ -21,6 +21,7 @@ const warningRules = [
   "react-dropzone-warn-upgrade",
   "table-warn-thExpandType",
   "tabs-warn-children-type-changed",
+  "warn-nav-flyouts-now-inline",
```

Comment:
Nit: just to align with other rule names (the `warn-key-codes-removed` may also be able to be updated)

```suggestion
  "nav-warn-flyouts-now-inline",
```

Reply:
Addressed.

Also addressed warn-key-codes-removed, which ended up raising an error rather than a warning anyways so I adjusted the rule name and description.